qualtran/bloqs/chemistry/pbc/first_quantization/first_quanization_test.py

```diff
@@ -1,19 +0,0 @@
-#  Copyright 2023 Google LLC
-#
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-#
-#      https://www.apache.org/licenses/LICENSE-2.0
-#
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
-
-from qualtran.testing import execute_notebook
-
-
-def test_notebook():
```

Comment:
?

Reply:
I've moved the first_quantization.py to select_and_prepare.py (the notebook is tested there.)